app/rest/metrics-collector/pom.xml

```diff
@@ -0,0 +1,226 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (C) 2016 Red Hat, Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>syndesis-rest-parent</artifactId>
+    <groupId>io.syndesis</groupId>
+    <version>1.2-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  
+  <artifactId>metrics-collector</artifactId>
+  <name>REST :: Metrics Collector</name>
+
+  <dependencies>
+
+    <!-- === Internal dependencies (don't touch without discussion) ========================== -->
+
+    <dependency>
+      <groupId>io.syndesis</groupId>
+      <artifactId>core</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>io.syndesis</groupId>
+      <artifactId>model</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>io.syndesis</groupId>
+      <artifactId>controllers</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>io.syndesis</groupId>
+      <artifactId>jsondb</artifactId>
```

Comment:
Do you require a direct DB dependency on jsondb here ? I really would like to go for *every DB access through the DAOs*.  Why else would we have then a DAO layer to abstract the DB implementation ? I have a lenghty discussion around this in #1054 around this and we ended up that we at least indicated clearly that the implementation is tight to JSONDB. 

I won't repeat myself here, all arguments are on the table (see #1054)

Reply:
@KurtStam I really want to know you reasoning here why to include jsondb. Sorry, ignoring this is just not fair and absolutely counterproductive. I invested a lot of efforts to collect arguments that adding dependencies like this at will and only because its easy *now* will destroy any architectural concept we have for the backend. 

Happy to argue with you and find a compromise, but this is just bad style.

Reply:
@rhuss the reason is two-fold. Firstly I'm using the jsonDBs specific API to select all RawMetrics for an integration based on the 'path'. Secondly it's now  easy to keep the RawMetrics in the metrics-collector module as it isn't part of our 'model'.  

Different story for the IntegrationSummaryMetrics for which I do use the DataManager.

Reply:
If I can't convince you that a hard dependency on jsondb is the wrong direction, then I can't convince you.

In order to reflect this live-and-die-together dependency I renamed the module to "rest-metrics-jsondb" to reflect this relation (same for "rest-logging-jsondb").